Patch from Void Linux. Here's the full message for the ff87-update, but
we're only taking the minidump_writer_linux part as it's the only
relevant one for Firefox on Wyverkiss.

From 9fd26c1bd4ecc5aa2d4336963ca2b0b2726bc5c8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=89rico=20Nogueira?= <erico.erc@gmail.com>
Date: Wed, 24 Mar 2021 09:59:36 -0300
Subject: [PATCH] firefox: update to 87.0.

minidump_writer_linux has issues on musl and doesn't support all
platforms Void does (thread_info doesn't touch ppc*), so we add a patch
to not build it at all. It seems to be a build system bug where
oxidized_breakpad can be enabled even when --disable-backtrace is set.
If next version still only enables it for x86_64, it might build/work
fine for us, see: https://github.com/msirringhaus/minidump_writer_linux/issues/4

diff --git a/Cargo.toml b/Cargo.toml
index 4076a60577..945f26f638 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,7 +13,6 @@ members = [
   "netwerk/test/http3server",
   "security/manager/ssl/osclientcerts",
   "testing/geckodriver",
-  "toolkit/crashreporter/rust_minidump_writer_linux",
   "toolkit/library/gtest/rust",
   "toolkit/library/rust/",
   "toolkit/mozapps/defaultagent/rust",
@@ -79,7 +78,6 @@ spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu4" }
 # failure's backtrace feature might break our builds, see bug 1608157.
 failure = { git = "https://github.com/badboy/failure", rev = "64af847bc5fdcb6d2438bec8a6030812a80519a5" }
 failure_derive = { git = "https://github.com/badboy/failure", rev = "64af847bc5fdcb6d2438bec8a6030812a80519a5" }
-minidump_writer_linux = { git = "https://github.com/msirringhaus/minidump_writer_linux.git", rev = "9191af36343846b2c7ada65b9602b481b717c4d8" }
 
 [patch.crates-io.cranelift-codegen]
 git = "https://github.com/mozilla-spidermonkey/wasmtime"
diff --git a/toolkit/moz.configure b/toolkit/moz.configure
index 806576b1ef..43ef8309f1 100644
--- a/toolkit/moz.configure
+++ b/toolkit/moz.configure
@@ -2487,8 +2487,6 @@
 # ==============================================================
 @depends(target)
 def oxidized_breakpad(target):
-    if target.kernel == "Linux" and target.os != "Android":
-        return target.cpu in ("x86", "x86_64")
     return False
 
 
